### ISSUE TITLE
Firefox pdf scrollbar looks horrible unlike Chrome

### ISSUE THREAD

**Issue by KarimRepo** [stephanrauh/ngx-extended-pdf-viewer#219]
Thank you for your PDF viewer component. Il works great on my Angular web application.

I have a little remark. I noticed that there is a difference between the aspect of the scrollbar in different browers. Firefox pdf scrollbar looks horrible unlike Chrome (I attached capture screens for this behavior). is it possible to use the perfect-scrollbar plugin (https://www.npmjs.com/package/perfect-scrollbar) to manage the default scroller and unifying the look in the different browers.

Thank you for your work. 

**Version info**
- Version of ngx-extended-pdf-viewer : 2.0.0-alpha.13

**Desktop**
 - OS: Windows 10 PRO
 - Browser : Chrome v80.0.3987.106 (Nice), Firefox v73.0.1 (Ugly)

**Captured Screens**
![scroller chrome](https://user-images.githubusercontent.com/10249071/75233057-8c707280-57b8-11ea-99d8-fd181d4f93ec.png)
![scroller firefox](https://user-images.githubusercontent.com/10249071/75233065-8ed2cc80-57b8-11ea-9a91-9bd1c8e0029d.png)

**Comment by stephanrauh** [comment]
Like so often, beauty is in the eye of the beholder. :) I can only guess: you prefer the small scrollbar?

I've tested the PDF viewer on https://pdfviewer.net on 

- Chrome and Firefox on OSX
- Chrome, Firefox, and Edge on Windows 10.

The scrollbars look differently depending on the operation system, but on the same OS, the scrollbars look similar on each browser. I didn't observe the huge difference your screenshots show. So I'm confused. Did you customize your OS or your Chrome?

**Comment by stephanrauh** [comment]
I can see the advantages of `PerfectScrollbar`, but I don't want to enforce it. Developers should be able to choose between the standard and the perfect scrollbar.

Maybe I'm just a bit tired, but at the moment I don't know how to implement that. Does something like

```typescript
private ps: any = undefined;

if (PerfectScrollbar) {
  ps = new PerfectScrollbar(container);
}
```

IMHO that's fine JavaScript code, but it doesn't compile in TypeScript. @KarimRepo Any clue?

**Comment by KarimRepo** [comment]
Thank you stephanrauh for your replay. I agree totally with you (beauty is in the eye of the beholder) :).
Actually, you helped me to temporary fix my problem.

public scrollbar: any = undefined;
constructor(@Inject(PERFECT_SCROLLBAR_CONFIG) public config: PerfectScrollbarConfigInterface) {}
ngAfterViewInit(): void {
    const container = document.querySelector('#viewerContainer');
    this.scrollbar = new PerfectScrollbar(container, this.config);
});

// in this case, config is not mandatory. it is just my configuration of the perfect scrollbar.

to answer your question, no I did not customize my OS or my Chrome.

Also, i think it is better to use the Angular version of the Perfect Scrollbar (https://www.npmjs.com/package/ngx-perfect-scrollbar).

**Comment by stephanrauh** [comment]
@KarimRepo I've added your solution to the showcase. However, I need your help. Sometimes the perfect scrollbar works, sometimes it doesn't. First impression: it works on first load, but it doesn't work after navigating to the page.

What am I doing wrong?

Here's the demo: https://pdfviewer.net/perfect-scrollbar

**Comment by pedrojrivera** [comment]
I am going to chime in as we also implemented ngx-perfect scrollbar a while back.

If you switch from panning to text selection, perfect scroll bar works on desktop as expected. 
On mobile, if you drag the perfect scroll bar in text selection mode, the scroll via touch scrolling misbehaves. 

So we have a web app that is used primarily for mobile and we had to remove the scrollbar. 

For our desktop apps we use it.
